content/en/docs/Tasks/creating-an-index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Creating an Index of operator bundles"
+weight: 3
+description: >
+  Add/Remove a collection of bundles to/from an Index
+---
+
+## Prerequisites 
+
+- [opm](https://github.com/operator-framework/operator-registry/releases) `v1.14.0+`
+- [docker](https://docs.docker.com/install/) version `17.03`+ or [podman](https://github.com/containers/libpod/blob/master/install.md) `v1.2.0+` or [buildah](https://github.com/containers/buildah/blob/master/install.md) `v1.7+`.
+
```

Comment:
IMO we need to first explain how to create the bundle with SDK, then we share that with opm and manually if that is the case.

Reply:
@camilamacedo86 the `Creating Operator Manifests` has a link to the sdk CSV generation doc. This document is about creating an index